tests/handlers/test_typing.py

```diff
@@ -114,16 +92,16 @@ def prepare(self, reactor, clock, hs):
             "retry_interval": 0,
             "failure_ts": None,
         }
-        self.datastore.get_destination_retry_timings.return_value = defer.succeed(
-            retry_timings_res
+        self.datastore.get_destination_retry_timings = Mock(
+            return_value=defer.succeed(retry_timings_res)
         )
```

Comment:
It is probably better to generate the actual data so each of these returns something expected, but not sure how involved that would be.